src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -123,6 +124,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       <IncludeAssets>Runtime;Native</IncludeAssets>
     </Reference>
 
+    <Reference Include="runtime.$(RuntimeIdentifier).microsoft.netcore.app" PrivateAssets="All" />
```

Comment:
Can someone sanity check my "fix" here?

Reply:
Is `runtime.*.microsoft.netcore.app` a shipping package?

Reply:
https://www.nuget.org/packages/runtime.win-x64.Microsoft.NETCore.App/3.0.0-preview3-27503-5

I added it here since it's where we get our crossgen tool from. I marked it as private assets all so that it doesn't affect the actual package that is being built.